jupyterhub/templates/image-puller/_helper.yaml

```diff
@@ -30,6 +30,12 @@ spec:
         component: {{ .name }}
         release: {{ .top.Release.Name }}
         heritage: {{ .top.Release.Service }}
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      # Arbitrarily large number, since we want all the old daemonsets to die immediately
+      # and new ones to be born
+      maxUnavailable: 1000
```

Comment:
We can set this to be 100% instead I think:
https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#rollingupdatedaemonset-v1beta2-apps

Reply:
Great catch, @consideRatio! fixed.